avaframe/ana3AIMEC/ana3AIMEC.py

```diff
@@ -430,14 +504,14 @@ def makeTransfoMat(rasterTransfo):
                 new coord system in the cross direction
     """
     w = rasterTransfo['domainWidth']
-    cellsize = rasterTransfo['cellsize']
+    csz = rasterTransfo['cellsize']
```

Comment:
mh, cellsize should be `cellSize` but I don't know whether this is too much for this PR, maybe open a new issue?

Reply:
made an issue